datalad/core/local/run.py

```diff
@@ -454,24 +454,16 @@ def format_command(dset, command, **kwds):
 
 
 def _execute_command(command, pwd, expected_exit=None):
-    from datalad.cmd import Runner
+    from datalad.cmd import WitlessRunner
 
     exc = None
     cmd_exitcode = None
-    runner = Runner(cwd=pwd)
+    runner = WitlessRunner(cwd=pwd)
     try:
         lgr.info("== Command start (output follows) =====")
         runner.run(
-            command,
-            # immediate output
-            log_online=True,
-            # not yet sure what we should do with the command output
-            # IMHO `run` itself should be very silent and let the command talk
-            log_stdout=False,
-            log_stderr=False,
-            expect_stderr=True,
-            expect_fail=True,
-            # TODO stdin
+            # command is always a string
+            ['sh', '-c', command],
```

Comment:
why not `/bin/sh`? (unlikely any silly soul gets `sh` into their PATH but you never know)... moreover subprocess in python uses `/bin/sh` according to docs:
```
   On POSIX with ``shell=True``, the shell defaults to :file:`/bin/sh`.  If
```

Reply:
Yes, that was the point here. `sh` should work fine on unix, but also works on windows. I will have to check whether `/bin/sh` would work too, or whether switching logic needs to be introduced.

In any case, this change would need to be done throughout the code base and not just in this PR.

Reply:
Confirmed that `/bin/sh` does **not** work on windows/conda

Reply:
I have implemented platform-shell execution support for WitlessRunner instead. https://github.com/datalad/datalad/pull/4996/commits/95f84a5dcd894b5ca05fea0bb72330be3c941af8